src/components/shared/Menu/index.jsx

```diff
@@ -0,0 +1,59 @@
+import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+import { OuterSpace, StyledMenu } from './menu.styles';
+
+/**
+ * Renders Anchored Menu.
+ *
+ * @param {Object} props The component props.
+ * @param {Object} props.anchorRef anchor element reference.
+ * @param {boolean} props.open state of the menu.
+ * @param {children} props.children JSX.Element.
+ * @param {Function} props.onClose onClose handler.
+ * @param {boolean} [props.sticky=false] is anchor sticky.
```

Comment:
Can we change to this
```suggestion
 * @param {Object} anchorRef anchor element reference.
 * @param {boolean} open state of the menu.
 * @param {children} children JSX.Element.
 * @param {Function} onClose onClose handler.
 * @param {boolean} [sticky=false] is anchor sticky.
```